core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -703,8 +703,15 @@ public void configure() throws ConfigurationException {
 							ConfigurationWarnings.add(this, log, "has a transaction timeout ["+getTransactionTimeout()+"] which exceeds the system transaction timeout ["+stt+"]");
 						}
 					}
+					String maximumTransactionTimeout = Misc.getMaximumTransactionTimeout();
```

Comment:
Could you please change Misc.getMaximumTransactionTimeout() and Misc.getSystemTransactionTimeout() in returning 'Integer' instead of 'String'?